### PR TITLE
Add few more content in containerized OpenShift

### PR DESCRIPTION
**Pull request by prabhatkumarthakur**
Signed-off-by: Prabhat Kumar Thakur prabhat.thakur@mayadata.io

**What this PR does / why we need it**:
This PR is to add more description about Containrized OpenShift installation.

Signed-off-by: Prabhat Kumar Thakur prabhat.thakur@mayadata.io

### CODE REVIEW

**Comment by prabhatkumarthakur** [comment]
@mahebbar

File: k8s/openshift/byo/baremetal/containerized_openshift_readme.md [hunk 1]
```diff
@@ -155,10 +155,59 @@ openshift_disable_check=disk_availability,memory_availability,docker_storage,doc
 20.10.45.111  openshift_public_ip=20.10.45.111 openshift_ip=20.10.45.111 openshift_public_hostname=osnode1.mdataqa.in openshift_hostname=osnode1.mdataqa.in containerized=True connect_to=20.10.45.111 ansible_connection=local
 ```
 
--While installing, if you get following error, use docker pull command to download the package on both master and nodes.
+-With above inventory file it will try to pull the latest openshift images. If you want to install openshift specific version such as v3.9.33 then you need to provide the image tag also in inventory file as mentioned below.
```
Comment (mahebbar) [inline k8s/openshift/byo/baremetal/containerized_openshift_readme.md]: - With the above... OpenShift... install a specific OpenShift version...v3.9.33, you must also provide the image tag in the inventory file as mentioned below.

File: k8s/openshift/byo/baremetal/containerized_openshift_readme.md [hunk 2]
```diff
@@ -155,10 +155,59 @@ openshift_disable_check=disk_availability,memory_availability,docker_storage,doc
...
+20.10.151.251  openshift_public_ip=20.10.151.251 openshift_ip=20.10.151.251 openshift_public_hostname=osc1.mayacb.in openshift_hostname=osc1.mayacb.in containerized=True ansible_connection=local
+
+[nfs]
+20.10.151.251  openshift_public_ip=20.10.151.251 openshift_ip=20.10.151.251 openshift_public_hostname=osc1.mayacb.in openshift_hostname=osc1.mayacb.in containerized=True ansible_connection=local
 
+[etcd]
+20.10.151.251  openshift_public_ip=20.10.151.251 openshift_ip=20.10.151.251 openshift_public_hostname=osc1.mayacb.in openshift_hostname=osc1.mayacb.in containerized=True ansible_connection=local
 ```
 
+Once the inventory file is created run the **prerequistes.yml** playbook using your inventory file
```
Comment (mahebbar) [inline k8s/openshift/byo/baremetal/containerized_openshift_readme.md]: ... is created, run ...

File: k8s/openshift/byo/baremetal/containerized_openshift_readme.md [hunk 3]
```diff
@@ -155,10 +155,59 @@ openshift_disable_check=disk_availability,memory_availability,docker_storage,doc
...
+
+```
+ansible-playbook -i <inventory_file> /usr/share/ansible/openshift-ansible/playbooks/prerequisites.yml
+```
+
+Run the **deploy_cluster.yml** playbook using your inventory file:
```
Comment (mahebbar) [inline k8s/openshift/byo/baremetal/containerized_openshift_readme.md]: ... file using the following command.

File: k8s/openshift/byo/baremetal/containerized_openshift_readme.md [hunk 4]
```diff
@@ -155,10 +155,59 @@ openshift_disable_check=disk_availability,memory_availability,docker_storage,doc
...
+
+```
+ansible-playbook -i <inventory_file> /usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.yml
+```
+
+-While installing, if you get following error, use docker pull command to download the package on both master and nodes.
```
Comment (mahebbar) [inline k8s/openshift/byo/baremetal/containerized_openshift_readme.md]: - While installing, if you get the following error, use the docker pull command on both the master and nodes.

**Comment by kmova** [comment]
the content will be made available in the OpenEBS docs. 

cc: @ranjithwingrider @prabhatkumarthakur

**Pull request closed without merging**